docs/development/creating-editing-renovate-presets.md

```diff
@@ -27,7 +27,7 @@ We have multiple kinds of `group:` presets, with different rules.
 ##### Rules for `group:*` presets
```

Comment:
There are still mentions of `config:base` in these files:

- [ ] `docs/usage/reading-list.md`
- [ ] `docs/usage/key-concepts.md`